docs/Usage-Guide.md

```diff
@@ -27,23 +27,83 @@ After visiting that URL on a smartphone, users will see an installation dialog e
 
 ## Using the Chatbot
 
-After opening the PWA, if a user is not signed in yet, the admin application users with a role that includes **View** permission for the Aira Unified application can sign in by clicking the **Continue with XbyK** button, which redirects them to the Xperience native administration sign-in page within the PWA.
+### Sign-in Process
+
+After opening the PWA, if a user is not signed in yet, they will see the sign-in screen. Users with appropriate permissions can sign in by:
+
+1. Clicking the **Continue with XbyK** button
+2. Entering their Xperience admin credentials
+3. Completing any required authentication steps (e.g., 2FA if enabled)
+4. Closing the administration interface after successful sign-in
 
-Users are expected to sign in and close the administration interface, as explained at the top of the screen.  
 ![Sign in](/images/AiraUnifiedSignIn.jpg)
 
-Aira Unified remembers the signed in user and the PWA will not require to repeat the sign in process.
+Aira Unified remembers the signed-in user and the PWA will not require repeating the sign-in process unless:
+- The user explicitly signs out
+- The session expires
```

Comment:
Can you add a note about session duration and automatic refresh if there is any?